clients/src/main/resources/common/message/ConsumerGroupDescribeRequest.json

```diff
@@ -18,7 +18,7 @@
   "type": "request",
   "listeners": ["broker"],
   "name": "ConsumerGroupDescribeRequest",
-  "validVersions": "0",
+  "validVersions": "0-1",
```

Comment:
Could you please add a note about version 1 here? Take a look at other requests/responses to see how we usually do it.